Doc/library/urllib.parse.rst

```diff
@@ -50,12 +50,16 @@ URL Parsing
 The URL parsing functions focus on splitting a URL string into its components,
 or on combining URL components into a URL string.
 
-.. function:: urlparse(urlstring, scheme='', allow_fragments=True)
+.. function:: urlparse(urlstring, scheme=None, allow_fragments=True, *, missing_as_none=False)
```

Comment:
I am really sorry, that I am contributing so late; but "missing_as_none=False" is confusing and not intuitive at all to me.

Pretty sure, others who have not participated are going to feel the same.


The function signature and term is not giving a signal on what it is meant to be. 

Are you open to new name?

Reply:
I am adding @gpshead, as one of the active developers in this area, to get his opinion too.

Reply:
What are your thoughts about `missing_as_empty` with the opposite semantic? In future, None will be returned for not defined components by default, and you will need to specify `missing_as_none=False` or `missing_as_empty=True` to restore the current behavior.